modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/RankEvalResult.java

```diff
@@ -31,43 +31,49 @@
  * For each precision at n computation the id of the search request specification used to generate search requests is returned
  * for reference. In addition the averaged precision and the ids of all documents returned but not found annotated is returned.
  * */
+// TODO do we need an extra class for this or it RankEvalResponse enough?
```

Comment:
The RankEvalResponse currently is just a wrapper around the RankEvalResult, maybe we can simply get rid of this class and move the fields etc... to the response class.